src/components/creator/Editor/SceneEdition/GridOptions/GridOptions.tsx

```diff
@@ -13,13 +14,16 @@ export const GridOptions = (props: StyleGridProps) => {
     return <>
         <Stack>
             <PBCard sx={{justifyContent: "center"}}>
-                <Stack sx={{padding: theme.spacing(1)}}>
+                <Stack sx={{padding: theme.spacing(1), maxWidth: "200px"}}>
                     <SizeEditor setStyleGrid={props.setStyleGrid} />
                     <ScenarioEditionButtons/>
                 </Stack>
             </PBCard>
-            <StatementEdition />
-            <ToolBoxDialog/>
+                <Stack sx={{flexDirection: "row"}}>
```

Comment:
```suggestion
                <Stack direction='row'>
```

Es como lo estamos haciendo en todos lados.

Reply:
perdon, no lo tenia... 
lo incorporo a mi "base de conocimientos"